<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gamingwidget</class>
 <widget class="QWidget" name="gamingwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>658</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLCDNumber" name="ScoreNumber">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="3">
      <widget class="QTextBrowser" name="box16">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="autoFormatting">
        <set>QTextEdit::AutoAll</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QTextBrowser" name="box11">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="autoFormatting">
        <set>QTextEdit::AutoAll</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QTextBrowser" name="box7">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="autoFormatting">
        <set>QTextEdit::AutoAll</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QTextBrowser" name="box5">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="autoFormatting">
        <set>QTextEdit::AutoAll</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTextBrowser" name="box6">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="autoFormatting">
        <set>QTextEdit::AutoAll</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QTextBrowser" name="box4">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="autoFormatting">
        <set>QTextEdit::AutoAll</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QTextBrowser" name="box13">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="autoFormatting">
        <set>QTextEdit::AutoAll</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QTextBrowser" name="box2">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="autoFormatting">
        <set>QTextEdit::AutoAll</set>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QTextBrowser" name="box15">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="autoFormatting">
        <set>QTextEdit::AutoAll</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QTextBrowser" name="box3">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="autoFormatting">
        <set>QTextEdit::AutoAll</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QTextBrowser" name="box1">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="autoFormatting">
        <set>QTextEdit::AutoAll</set>
       </property>
       <property name="lineWrapMode">
        <enum>QTextEdit::FixedPixelWidth</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QTextBrowser" name="box12">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="autoFormatting">
        <set>QTextEdit::AutoAll</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QTextBrowser" name="box9">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="autoFormatting">
        <set>QTextEdit::AutoAll</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QTextBrowser" name="box8">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="autoFormatting">
        <set>QTextEdit::AutoAll</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QTextBrowser" name="box10">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="autoFormatting">
        <set>QTextEdit::AutoAll</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QTextBrowser" name="box14">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="autoFormatting">
        <set>QTextEdit::AutoAll</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
